recipe/0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch

```diff
@@ -1,7 +1,7 @@
-From 258a0b764df7e085ab46785cab7b07ec52d07d4c Mon Sep 17 00:00:00 2001
+From 9a413b6425e0a8cc156b2f840b5c10617bbaba7f Mon Sep 17 00:00:00 2001
 From: William Jamir Silva <williamjamir@gmail.com>
```

Comment:
I wrote these patches. Please correct this misattribution.

Reply:
@mingwandroid I guess that the patch was refreshed by William and that header was overwritten. I don't think anyone was trying to steal you credit. We can definitely fix that in a new iteration of this packages.

Reply:
I'll do that while I'm at it, I need to rebase with some more fixes any way.

Reply:
Hey @ocefpaf, while I do not care about credit for the sake of it, others might. My concern is that outwardly, it would appear that William Jamir Silva wrote this and therefore can explain what does and why. I'm not sure that's the case, but regardless, any unexpected change to credit lines in patches should, IMHO trigger a linting failure (with a way to say, yes it is OK).

Reply:
BTW I consider these moral rights of patch authors at least as important as correct licence identification, perhaps more so, because a contributor *could* be offended to the level that they stop contributing when this happens.